Library/Formula/brew-file.rb

```diff
@@ -0,0 +1,32 @@
+class BrewFile < Formula
+  desc "Brewfile manager for Homebrew."
+  homepage "https://github.com/rcmdnk/homebrew-file/"
+  url "https://github.com/rcmdnk/homebrew-file.git",
```

Comment:
Are there git submodules here? If not we should use the archive tarball & `sha256` rather than a tag & revision.